package.json

```diff
@@ -0,0 +1,145 @@
+{
+  "name": "terra-application",
+  "main": "lib/application-base/ApplicationBase.js",
+  "version": "1.11.0",
+  "description": "A framework to support application development with Terra components",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-application"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-application",
+    "Application",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-application/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra",
+    "overrides": [
+      {
+        "files": [
+          "src/terra-dev-site/**/*.jsx"
+        ],
+        "rules": {
+          "import/no-extraneous-dependencies": "off",
+          "import/no-unresolved": "off",
+          "import/extensions": "off"
+        }
+      },
+      {
+        "files": [
+          "*.doc.jsx"
+        ],
+        "rules": {
```

Comment:
Why are these rules disabled?

Reply:
Those got pulled in from dev-site, since we're not doing any code splitting I can remove this block.

Reply:
I lied, the linter throws a fit about the raw-loader imports with these rules enabled..